assignments/lambda-classes.js

```diff
@@ -1 +1,189 @@
 // CODE here for your Lambda Classes
+//first commit
+
+class Person {
+  constructor(things) {
+    this.name = things.name;
+    this.location = things.location;
+    this.age = things.age;
+    this.gender = things.gender;
+  }
+  speak() {
+    console.log(`Hello, my name is ${this.name}, I am from ${this.location}`)
+  }
+}
+
+class Instructor extends Person {
+  constructor(stuff) {
```

Comment:
```suggestion
  constructor(props) {
```
I recommend just using props here, as technically you're passing down the "properties" of it's parent class. This will get you used to the pattern in React, too.